packages/quantic/force-app/main/default/lwc/quanticSvg/default.html

```diff
@@ -0,0 +1 @@
+<template></template>
```

Comment:
Do you need a default one?

Reply:
I prefer always returning some default value for my switch case. What do you think?

Reply:
good point, it's just 1 more file necessary for that. It's fine, I was asking your opinion